app/assets/javascripts/discourse/tests/integration/components/discovery/filter-tips-test.gjs

```diff
@@ -0,0 +1,333 @@
+import didInsert from "@ember/render-modifiers/modifiers/did-insert";
```

Comment:
Might be worth adding a bit more to https://github.com/discourse/discourse/blob/6db9ab44bbb428b71d72fd30e328f8c97b3505ec/spec/system/filtering_topics_spec.rb#L3-L3 , but overall this covers a lot :)